src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutes.scala

```diff
@@ -62,15 +62,14 @@ trait UserRoutes extends UserInfoDirectives with SamRequestContextDirectives {
                 }
               }
             }
-          }
-        } ~ withSamRequestContext { samRequestContext =>
+        } ~
           pathPrefix("termsofservice") {
             pathPrefix("status") {
               pathEndOrSingleSlash {
                 get {
-                  requireActiveUser(samRequestContext) { samUser =>
+                  requireUserAllowInactive(samRequestContext) { samUser =>
```

Comment:
Seems to me like changing from `requireActiveUser` to `requireUserAllowInactive` makes sense here in that an inactive user should be able to interrogate their ToS status, just want to make sure this was intentional?  I'll also ask if this is tested?  (I'll come back and delete this if I find the test later in the review)